lib/quby/questionnaires/entities/charting/chart.rb

```diff
@@ -10,9 +10,20 @@ class Chart
           attribute :key,                        Symbol
           attribute :title,                      String
           attribute :plottables,                 Array
-          # if categories are defined, plottable values should correspond to values from this array
-          # and the graph will  be plotted with matching y axis categories
+          # if y_categories are defined, plottable values should correspond to values from this array
+          # and the graph will be plotted with corresponding y axis categories
+          # example (icl_r): ["Zeer laag", "Laag", "Gemiddeld", "Hoog", "Zeer Hoog"] (caution, capitalization oddity)
+          # NB: only implemented for bar charts on the roqua side
           attribute :y_categories,               Array
+          # if y_range_categories are defined, plottable values should fall in the ranges that compose the keys of this
+          # hash. The chart will label these ranges of y_values with the corresponding value in the hash on the y axis.
+          # example:
+          # {(0..30) => "Zeer laag", (30..40) => "Laag", (40..60) => "Gemiddeld",
+          # (60..70) => "Hoog", (70..100) => "Zeer hoog"}
```

Comment:
Dat de questionnaires al geschreven zijn is sowieso een probleem. We gaan immers nu niet al dsl bouwen om de generieke oplossing te ondersteunen, dus meer werk aan de achterkant doen om dat alvast te ondersteunen heeft toch ook geen zin?

De simpele hash is voor graph y_range_categories prima, het is een 1 op 1 relatie met hoe plotbands werken. Voor generieke score categories zou ik inderdaad een object bouwen die ook netjes geexporteerd kan worden, maar voor grafiekconfiguratie is dat niet van toepassing.

Exclusive ranges icm last gebruiken lijkt mij wel een goed idee.

Ik ben benieuwd hoe jij dit ticket zou hebben ingevuld dus daar moeten we het irl nog even over hebben.